lib/logger/lib/logger/backends/console.ex

```diff
@@ -77,10 +77,10 @@ defmodule Logger.Backends.Console do
       format_event(level, msg, ts, md, state)
       |> color_event(level, colors)
     try do
-      IO.write(device, output)
+      send(device, {:io_request, self(), self(), {:put_chars, :unicode, output}})
     rescue
       ArgumentError ->
```

Comment:
Thank you! However, this clause will never be raised now. Can you try the following variation instead of the `try do`:

case :unicode.characters_to_binary(output) do
  {:error, good, bad} ->
    send(device, {:io_request, self(), self(), {:put_chars, :unicode, [good | Logger.Formatter.prune(bad)]}})
  good ->
    send(device, {:io_request, self(), self(), {:put_chars, :unicode, good}})
end
```

And let us know how it performs CPU and memory wise?